intermediate_source/realtime_rpi.rst

```diff
@@ -0,0 +1,259 @@
+classification model in real time (30 fps+) on the CPU.
+
+This was all tested with Raspberry Pi 4 Model B 4GB but should work with the 2GB
+variant as well as on the 3B with reduced performance.
+
+.. image:: https://user-images.githubusercontent.com/909104/152895495-7e9910c1-2b9f-4299-a788-d7ec43a93424.jpg
+
+Prerequisites
+~~~~~~~~~~~~~~~~
+
+To follow this tutorial you'll need a Raspberry Pi 4, a camera for it and all
+the other standard accessories.
+
+* `Raspberry Pi 4 Model B 2GB+ <https://www.raspberrypi.com/products/raspberry-pi-4-model-b/>`_
+* `Raspberry Pi Camera Module <https://www.raspberrypi.com/products/camera-module-v2/>`_
+* Heat sinks and Fan (optional but recommended)
+* 5V 3A USB-C Power Supply
+* SD card (at least 8gb)
+* SD card read/writer
+
+
+Raspberry Pi 4 Setup
+~~~~~~~~~~~~~~~~~~~~~~~
+
+PyTorch only provides pip packages for Arm 64bit (aarch64) so you'll need to install a 64 bit version of the OS on your Raspberry Pi
+
+You can download the latest arm64 Raspberry Pi OS from https://downloads.raspberrypi.org/raspios_arm64/images/ and install it via rpi-imager.
+
+**32-bit Raspberry Pi OS will not work.**
+
+.. image:: https://user-images.githubusercontent.com/909104/152866212-36ce29b1-aba6-4924-8ae6-0a283f1fca14.gif
+
+Installation will take at least a few minutes depending on your internet speed and sdcard speed. Once it's done it should look like:
+
+.. image:: https://user-images.githubusercontent.com/909104/152867425-c005cff0-5f3f-47f1-922d-e0bbb541cd25.png
+
+Time to put your sdcard in your Raspberry Pi, connect the camera and boot it up.
+
+.. image:: https://user-images.githubusercontent.com/909104/152869862-c239c980-b089-4bd5-84eb-0a1e5cf22df2.png
+
+
+Once that boots and you complete the initial setup you'll need to edit the ``/boot/config.txt`` file to enable the camera.
+
+.. code:: toml
+
+    # This enables the extended features such as the camera.
+    start_x=1
+
+    # This needs to be at least 128M for the camera processing, if it's bigger you can just leave it as is.
+    gpu_mem=128
+
+    # You need to commment/remove the existing camera_auto_detect line since this causes issues with OpenCV/V4L2 capture.
+    #camera_auto_detect=1
+
+And then reboot. After you reboot the video4linux2 device ``/dev/video0`` should exist.
+
+Installing PyTorch and OpenCV
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PyTorch and all the other libraries we need have ARM 64-bit/aarch64 variants so you can just install them via pip and have it work like any other Linux system.
+
+.. code:: shell
+
+    $ pip install torch torchvision torchaudio
+    $ pip install opencv-contrib-python
+    $ pip install numpy --upgrade
+
+.. image:: https://user-images.githubusercontent.com/909104/152874260-95a7a8bd-0f9b-438a-9c0b-5b67729e233f.png
+
+
+We can now check that everything installed correctly:
+
+.. code:: shell
+
+  $ python3 -c "import torch; print(torch.__version__)"
+  1.10.0+cpu
```

Comment:
I'd remove this line so you don't have to keep updating tutorial with new version

Reply:
done